<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventOutputWriterDialog</class>
 <widget class="QWidget" name="EventOutputWriterDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Output Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QCheckBox" name="mCSV">
     <property name="text">
      <string>CSV</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="mEVT">
     <property name="text">
      <string>EVT</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="mOutputTypeSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mOutputTypeLabel">
     <property name="text">
      <string>event_output Type:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change the tabs to change what properties of event_output you are editing.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="mDummyTab">
      <attribute name="title">
       <string/>
      </attribute>
     </widget>
     <widget class="QWidget" name="mCSV_EventsTab">
      <attribute name="title">
       <string>csv_event_output Events</string>
      </attribute>
      <layout class="QGridLayout" name="mEventsTabLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="mCSV_EventsScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="wizard::EventOutput::event::Widget" name="mCSV_Events">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>554</width>
            <height>214</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change the events for the csv_event_output block.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mCSV_SettingsTab">
      <attribute name="title">
       <string>csv_event_output Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QScrollArea" name="mCSV_SettingsScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="wizard::EventOutput::setting::Widget" name="mCSV_Settings">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>554</width>
            <height>214</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change the settings for the csv_event_output block.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mEVT_EventsTab">
      <attribute name="title">
       <string>event_output Events</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QScrollArea" name="mEVT_EventsScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="wizard::EventOutput::event::Widget" name="mEVT_Events">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>554</width>
            <height>214</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change the events for the event_output block.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mEVT_SettingsTab">
      <attribute name="title">
       <string>event_output Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QScrollArea" name="mEVT_SettingsScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="wizard::EventOutput::setting::Widget" name="mEVT_Settings">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>554</width>
            <height>214</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change the settings for the event_output block.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="wizard::EventOutput::FileLocatorWidget" name="mFile" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="wizard::EventOutput::DialogButtonsWidget" name="mMainButtons" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wizard::EventOutput::DialogButtonsWidget</class>
   <extends>QWidget</extends>
   <header>DialogButtonsWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wizard::EventOutput::event::Widget</class>
   <extends>QWidget</extends>
   <header>EventWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wizard::EventOutput::FileLocatorWidget</class>
   <extends>QWidget</extends>
   <header>FileLocatorWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wizard::EventOutput::setting::Widget</class>
   <extends>QWidget</extends>
   <header>SettingWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
